src/pages/[platform]/build-a-backend/graphqlapi/modify-amplify-generated-resources/index.mdx

```diff
@@ -1,19 +1,10 @@
 import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
-                  
+
 export const meta = {
-  title: 'Override Amplify-generated AppSync resources',
-  description: 'The "amplify override api" command generates a developer-configurable "overrides" TypeScript file that provides Amplify-generated AppSync resources as CDK constructs. For example, developers can configure API settings that are not directly available in the Amplify CLI workflow, such as X-Ray tracing.',
-  platforms: [
-                'android',
-                'angular',
-                'flutter',
-                'javascript',
-                'nextjs',
-                'react',
-                'react-native',
-                'swift',
-                'vue'
-              ]
+  title: 'Set up Amplify GraphQL API',
```

Comment:
@renebrandel The above platforms would need to be listed if you want those visiting react etc to see the JS content